tests/test_schema_retrieval.py

```diff
@@ -25,23 +24,35 @@
 class TestSchemaRetrieval:
     "Test schema retrieval."
 
-    def test_custom_schema_local(self):
-        "Test retrieving user-specified local schema files."
+    @pytest.mark.parametrize('location_type',
+                             ('absolute_dir',
+                              'relative_dir',
```

Comment:
Do we need to feed in `relative_dir` as one of the parameters here since the assertions below are comparing against the result of `schema_file_relative_dir` anyway?

Reply:
It's just going through different path. We want to make sure the retrieval function can handle this properly